### PR TITLE
Add thirdyparty-licenses target

### LINKED ISSUE #72: Auto-generate third-party licenses
Goal: automatically generate https://github.com/DataDog/stratus-red-team/blob/main/LICENSE-3rdparty.csv

First step: Using https://github.com/google/go-licenses manually

Second step: Pre-commit hook

### PR DESCRIPTION
**Pull request by xen0ldog**
Fixes #72

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,7 +15,11 @@ docs:
 test:
 	go test ./... -v
 
+thirdparty-licenses:
+	go install github.com/google/go-licenses
```
Comment (christophetd) [inline Makefile:19]: this didn't work for me:

```
go install github.com/google/go-licenses
no required module provides package github.com/google/go-licenses; to add it:
	go get github.com/google/go-licenses
make: *** [thirdparty-licenses] Error 1
```

I would suggest either removing the `go install` from the Makefile target, either adding all the commands for it to work

File: Makefile [hunk 2]
```diff
@@ -12,7 +15,11 @@ docs:
...
+	~/go/bin/go-licenses csv github.com/datadog/stratus-red-team/cmd/stratus | sort > $(ROOT_DIR)/LICENSE-3rdparty.csv
```
Comment (christophetd) [inline Makefile:20]: You're making an assumption that GOPATH is `~/go`

Comment (xen0ldog) [reply Makefile]: Fixed to use $GOPATH

File: Makefile (hunk 1 above)
Comment (xen0ldog) [reply Makefile:20]: Added `go get`

**Pull request merged**